.github/workflows/rustls-rustcrypto.yml

```diff
@@ -57,14 +58,52 @@ jobs:
           - armv7a-none-eabi
           - thumbv7em-none-eabi
           - wasm32-unknown-unknown
+          # XXX XXX no atomic ptr
+          - riscv32imc-unknown-none-elf
+          - thumbv6m-none-eabi
     steps:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ matrix.rust }}
           targets: ${{ matrix.target }}
+      # XXX XXX QUICK BUILD WORKAROUND FOR no atomic ptr
+      - run: |
+          cargo add once_cell --no-default-features -F critical-section
+          cargo add portable-atomic --no-default-features -F critical-section
+          cargo add spin --no-default-features -F portable_atomic
+        if: ${{ matrix.target == 'riscv32imc-unknown-none-elf' }}
+      # XXX XXX MODIFIED BUILD WORKAROUND FOR no atomic ptr
+      - run: |
+          cargo add once_cell --no-default-features -F critical-section
+          cargo add portable-atomic --no-default-features -F unsafe-assume-single-core
+          cargo add spin --no-default-features -F portable_atomic
+        if: ${{ matrix.target == 'riscv32imc-unknown-none-elf' }}
```

Comment:
OOPS accidentally did both workarounds for the same target, I meant to add these 2 workarounds each one for a different target with no atomic ptr.

Build failed due to the invalid feature combination for `portable-atomic`.